### ISSUE TITLE
One-Time-Code support

### ISSUE THREAD

**Issue by au2001** [au2001/icloud-passwords-firefox#8]
iCloud Keychain supports setting up OTC and generating them.
A new tab for OTC could be added with the available codes.
SMS authentication codes should also be supported.

**Comment by nicholas-c** [comment]
@au2001 saw this was removed from the v1.4 milestone, is this still on the radar for support?

**Comment by au2001** [comment]
@nicholas-c Absolutely! Since this has been requested a lot, this is actually one of the next features I'm targeting after #13

**Comment by meszolym** [comment]
Hey! Although it seems to be there, it's not really working, it gives me totally different OTPs than the app, and they don't work. :(

![Image](https://github.com/user-attachments/assets/ba243dff-a903-4f27-9840-120959605eb7)

One the left, the code from the app, on the right, the code entered by the plugin. I'm on windows, Zen. Passwords work beautifully.

**Comment by nicholas-c** [comment]
@meszolym not the creator, but this repo is now deprecated I believe, Apple maintain their own official version now.

**Comment by meszolym** [comment]
@nicholas-c , I'm on Windows. Firefox / Zen does not work on windows sadly, only with this. Also, last commit was a month ago, I don't see how that means deprecated.

**Comment by nicholas-c** [comment]
@meszolym ah my bad! Sorry!

**Comment by meszolym** [comment]
@nicholas-c , No worries 😄

**Comment by meszolym** [comment]
'+ info: from what I can see, if I have two codes set up for a.x.y and b.x.y then it bugs out and gets the same one for both. It also happens with passwords from what I can see, so seemingly it only cares about the domain, not the subdomain.
Might actually be a bug in the Windows app itself: https://discussions.apple.com/thread/254873520?sortBy=rank
